### ISSUE TITLE
 Property 'species' does not exist on type 'SymbolConstructor'

### ISSUE THREAD

**Issue by Vivek205** [tradingview/charting-library-examples#78]
**Checklist**

- [ ] This issue is **NOT** related to the charting library itself

**Describe the bug**

Err occured while trying to run the reactjs-javascript example

**To Reproduce**

react-javascript example
1. npm run eject
2. remove ModuleScopePlugin from webpack config
3. npm install @types/jquery -D
4. npm start

**Expected behavior**

 To integrate the cchart and make it work

**Screenshots**

![image](https://user-images.githubusercontent.com/26110173/51340013-e67e0f00-1ab3-11e9-94e0-2e140e57b232.png)

**Additional context**

I exxamined the source code. the symbolConstructor is missing species
![image](https://user-images.githubusercontent.com/26110173/51340130-27762380-1ab4-11e9-8316-230a7d90f2b4.png)

**Comment by timocov** [comment]
> Err occured while trying to run the reactjs-javascript example

How TypeScript's code related to javascript example? It looks like problem of configuring your project. But you can try to fix your tsconfig file (add `es2015.symbol.wellknown` lib).

**Comment by Vivek205** [comment]
> How TypeScript's code related to javascript example?
**Long Story Short**:
The error started occuring when I updated the react-scripts to the latest 2.x.x version. 
It lead to too many errors. I tried a lot of fixes including `es2015.symbol.wellknown` . Still same error persists.

Now I started with a  fresh clone. 

1. Moved the charting_library folder inside src. 
2. I got the err 
```
./src/charting_library/charting_library.min.js
  Line 1:  'define' is not defined  no-undef
  Line 1:  'define' is not defined  no-undef
```
I followed the steps in [#36]( https://github.com/tradingview/charting-library-examples/issues/36) and inserted `/* eslint-disable */`  on the top of file **charting_library.min.d** .  But still I'm getting the same issue.
![image](https://user-images.githubusercontent.com/26110173/51396580-5ac4bb00-1b65-11e9-991b-adb8a4dbf6d3.png)

**PS**:   _If you could update the examples with new create-react-app it will be usefull. I feel that, there are some breaking changes in react-scripts leading to incompatibility_

**Comment by timocov** [comment]
>  on the top of file charting_library.min.d .

But the errors are in `charting_library.min.js` (js, not d.ts).

**Comment by Vivek205** [comment]
> 
> 
> > on the top of file charting_library.min.d .
> 
> But the errors are in `charting_library.min.js` (js, not d.ts).

Yeah previously it was with d.ts. But now I started with fresh clone. Executed the steps as in [the previous comment](https://github.com/tradingview/charting-library-examples/issues/78#issuecomment-455589595).

**Comment by timocov** [comment]
> I followed the steps in #36 and inserted /* eslint-disable */ on the top of file charting_library.min.d .

You need to insert `/* eslint-disable */` is js file.
